### PR TITLE
Update docs relating to score

### PR DESCRIPTION
**Pull request by nanaya**
Something like this? 🤷 

(I'm not touching the score object docs just yet)

### CODE REVIEW

File: app/Http/Controllers/BeatmapPacksController.php [hunk 1]
```diff
@@ -86,6 +86,8 @@ public function index()
      * user_completion_data |
      *
      * @urlParam pack string required The tag of the beatmap pack to be returned.
+     *
+     * @queryParam legacy_only integer Whether or not to consider lazer scores for user completion data. Defaults to 1. Example: 0
```
Comment (respektive) [inline app/Http/Controllers/BeatmapPacksController.php:90]: Doesn't it default to 0? I'm at least getting lazer scores without setting this param in the request and have to manually set it to 1 to exclude them.

Comment (notbakaneko) [reply app/Http/Controllers/BeatmapPacksController.php:90]: I think this should say it's also based on the user preference setting?

**Review by notbakaneko — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/BeatmapPacksController.php (hunk 1 above)
Comment (nanaya) [reply app/Http/Controllers/BeatmapPacksController.php]: user preference is not used for api request

File: resources/views/docs/info.md.blade.php [hunk 2]
```diff
@@ -124,6 +124,16 @@
 v2      | current
 v1      | _legacy api provided by the old site, will be deprecated soon_
 
+### API Response Version
+
+Sometimes API response needs updating in non-backward compatible ways. In such cases, `x-api-version` header is used to determine which version of the response will be returned.
```
Comment (notbakaneko) [inline resources/views/docs/info.md.blade.php:129]: forgot about this line 👀 
```suggestion
Sometimes, an API response need to be updated in non-backward compatible ways. In such cases, the `x-api-version` header is used to determine which version of the response will be returned.
```

**Review by notbakaneko — CHANGES_REQUESTED** [review]

**Review by notbakaneko — APPROVED** [review]

**Pull request merged**